core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -250,9 +250,9 @@ private static void extractSite(String dir, boolean force, Log log) throws IOExc
                 log.error().$("could not find site [resource=").$(publicZip).$(']').$();
             }
         }
-        copyConfResource(dir, force, buffer, "conf/date.formats", log);
-        copyConfResource(dir, force, buffer, "conf/mime.types", log);
-        copyConfResource(dir, force, buffer, "conf/server.conf", log);
+//        copyConfResource(dir, force, buffer, "conf/date.formats", log);
+//        copyConfResource(dir, force, buffer, "conf/mime.types", log);
+//        copyConfResource(dir, force, buffer, "conf/server.conf", log);
```

Comment:
Why commenting the code, should we just remove it?

Reply:
I would suggest to copy mime.types and not others; this file is highly unlikely to change

Reply:
done